Panic when `instrumentation-opt-in` enabled

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Output of `ddev debug test`

_No response_

### Expected Behavior

Don't throw an error.

### Actual Behavior

```bash
$ ddev version
 ITEM             VALUE                                         
 DDEV version     v1.22.0-alpha3-29-g9b3b0d9b                   
 architecture     amd64                                         
 db               ddev/ddev-dbserver-mariadb-10.4:v1.22.0-beta1 
 ddev-ssh-agent   ddev/ddev-ssh-agent:v1.22.0-beta1             
 docker           24.0.2                                        
 docker-compose   v2.18.1                                       
 docker-platform  dev                                           
 mutagen          0.17.1                                        
 os               linux                                         
 router           traefik:v2.10                                 
 web              ddev/ddev-webserver:v1.22.0-beta1             
 
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x1d8 pc=0x91d79b]

goroutine 1 [running]:
github.com/ddev/ddev/pkg/ddevapp.(*DdevApp).GetOmittedContainers(...)
        /home/runner/work/ddev/ddev/pkg/ddevapp/ddevapp.go:371
github.com/ddev/ddev/pkg/ddevapp.(*DdevApp).TrackProject(0x0)
        /home/runner/work/ddev/ddev/pkg/ddevapp/amplitude_project.go:36 +0x7b
github.com/ddev/ddev/cmd/ddev/cmd.glob..func48(0x10e72c0, {0x112cda0?, 0x0?, 0x0?})
        /home/runner/work/ddev/ddev/cmd/ddev/cmd/root.go:98 +0x86
github.com/spf13/cobra.(*Command).execute(0x10e72c0, {0x112cda0, 0x0, 0x0})
        /home/runner/work/ddev/ddev/vendor/github.com/spf13/cobra/command.go:876 +0x770
github.com/spf13/cobra.(*Command).ExecuteC(0x10ea4c0)
        /home/runner/work/ddev/ddev/vendor/github.com/spf13/cobra/command.go:974 +0x3bd
github.com/spf13/cobra.(*Command).Execute(...)
        /home/runner/work/ddev/ddev/vendor/github.com/spf13/cobra/command.go:902
github.com/ddev/ddev/cmd/ddev/cmd.Execute()
        /home/runner/work/ddev/ddev/cmd/ddev/cmd/root.go:152 +0x25
main.main()
        /home/runner/work/ddev/ddev/cmd/ddev/main.go:28 +0xc5
```

### Steps To Reproduce

1. Use the latest `ddev` binary.
2. `ddev config global --instrumentation-opt-in=true`
3. Run `ddev version` or `ddev config global`.

### Anything else?

_No response_

--- a/cmd/ddev/cmd/root.go
+++ b/cmd/ddev/cmd/root.go
@@ -95,7 +95,10 @@ Support: https://ddev.readthedocs.io/en/stable/users/support`,
 				instrumentationApp = app
 			}
 		}
-		instrumentationApp.TrackProject()
+
+		if instrumentationApp != nil {
+			instrumentationApp.TrackProject()
+		}
 
 		// TODO remove once Amplitude has verified with an alpha release.
 		// Do not report these commands

